pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -17,11 +18,14 @@ import (
 	"github.com/gorilla/mux"
 	"github.com/sirupsen/logrus"
 
+	"regexp"
+
 	"github.com/Azure/ARO-RP/pkg/api"
 	"github.com/Azure/ARO-RP/pkg/api/admin"
 	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
 	"github.com/Azure/ARO-RP/pkg/env"
 	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+	"github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/features"
```

Comment:
Need to fix the import ordering.  It goes as follows:

1. golang standard library

2. third party imports (i.e. github.com/gorilla/mux) 

3. first party imports (i.e. github.com/Azure/ARO-RP/*)